tests/manifest.ttl

```diff
@@ -1,17 +1,27 @@
-## Distributed under both the "W3C Test Suite License" [1]
-## and the "W3C 3-clause BSD License".
 ## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
 ## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
 
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests#>
+@PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
```

Comment:
`PREFIX`, no '.'.

i.e. RDF 1.1 style Turtle.

Reply:
I also prefer the `PREFIX …` to the `@prefix … .` notation.

Further, I suggest ordering the list by the prefix (most "alpha sort lines" tools will do the right thing) -- because it helps avoid duplicate declarations, and makes human review of the file easier.